app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -138,6 +138,7 @@ public enum SettingsEnum {
     HIDE_SHORTS_COMMENTS_BUTTON("revanced_hide_shorts_comments_button", BOOLEAN, FALSE),
     HIDE_SHORTS_REMIX_BUTTON("revanced_hide_shorts_remix_button", BOOLEAN, FALSE),
     HIDE_SHORTS_SHARE_BUTTON("revanced_hide_shorts_share_button", BOOLEAN, FALSE),
+    HIDE_SHORTS_INFO_PANEL("revanced_hide_shorts_info_panel", BOOLEAN, FALSE),
```

Comment:
It may be true by default. Currently, everything is false for the sole reason, including any filter patch, it also includes all filters and causes unintended patches. A way to solve this as elegantly as possible would be to make the `LithoFilter` patch inject separate hooks for each type of filter patch. This would solve the current (big) problem of the litho patches being inseparable.

Reply:
Yes, every filter can be enabled or disabled by default. The problem I mention here is regarding the following situation:

You include a <filter> patch such as "hide-ads". You expect it to hide ads. For some reason, now shorts components are hidden. The reason for this is, that including the "hide-ads" patch, causes LithoFilterPatch to inject a hook and thus enable all filters. An elegant solution is mentioned in my previous comment.